modules/user/client-react/signUp/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { NavLink, withRouter } from 'react-router-dom';
+
+import { translate } from '@restapp/i18n-client-react';
+import { MenuItem } from '@restapp/look-client-react';
+import ClientModule from '@restapp/module-client-react';
+
+import Login from './containers/Login';
+import Register from './containers/Register';
+import ForgotPassword from './containers/ForgotPassword';
+import ResetPassword from './containers/ResetPassword';
+import signUpReducer from './reducers';
+import resources from './locales';
+
+import { AuthRoute, IfLoggedIn, IfNotLoggedIn, withLogout, WithLogoutProps } from '../containers/Auth';
+
+export interface LoginSubmitProps {
```

Comment:
I suggest using `types` dir for not polluting `index` files.

Reply:
Done